test/unionFind.spec.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Created by tatsiak on 4/1/17.
+ */
+ 'use strict'
+ const algolib = require('../index.js');
+ 
+ 
+ describe.only('#unionFind', function () {
```

Comment:
forgot `only`
be careful, if you break something in the code, no one will see it